packages/core/src/components/accordion/accordion-item/accordion-item.tsx

```diff
@@ -21,33 +21,40 @@ export class TdsAccordionItem {
   @Prop() disabled: boolean = false;
 
   /** Set to true to expand panel open */
-  @Prop() expanded: boolean = false;
+  @Prop({ mutable: true, reflect: true }) expanded: boolean = false;
 
   /** When true, 16px on right padding instead of 64px */
   @Prop() paddingReset: boolean = false;
 
+  /** Fires when the Accordion Item is clicked, but before it is closed or opened. */
+  @Event({
+    eventName: 'tdsToggle',
+    composed: true,
+    cancelable: true,
+    bubbles: true,
+  })
+  tdsToggle: EventEmitter<{
+    expanded: boolean;
+    header: string;
```

Comment:
Might wanna make this an optional param? Otherwise it could be considered a breaking change.

Reply:
You are right!